CODEOWNERS

```diff
@@ -1 +1 @@
-* @projectsyn/tarazed
+* @projectsyn/aldebaran
```

Comment:
Core Project Syn isn't currently owned by Aldebaran, so this change doesn't make sense to me.